backend/package.json

```diff
@@ -40,7 +40,6 @@
     "joi": "^14.3.1",
     "mongodb": "^3.1.13",
     "mongoose": "^5.4.14",
-    "mongoose-data-seed": "^1.0.7",
```

Comment:
Is `package.json` paremovinai, bet `package-lock.json` liko visi change'ai del sito package :D Reiktu ir `package-lock.json` sutaisyt.

Reply:
No worries, sutvarkiau uz tave su 4355711635158411620f0923970702974ff79889